src/components/popover-container/popover-container.component.tsx

```diff
@@ -376,6 +377,7 @@ export const PopoverContainer = forwardRef<
       ref: openButtonRef,
       "aria-label": openButtonAriaLabel || title,
       id: isOpen ? undefined : popoverContainerId,
+      className: "persist-primary-styling",
```

Comment:
comment (non-blocking): as we use `primary` as a `buttonType`/`variant` this className might suggest something unintended. Maybe something like `persist-original-styling`